server/lib/404.js

```diff
@@ -6,6 +6,7 @@
 
 // It's a 404 not found response.
 
+/*jshint unused: false */
 module.exports = function (req, res, next) {
```

Comment:
I've found unused argument detection to be useful. Let's stick with jshint exemptions, because cases like this are the exception.